core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -147,20 +140,16 @@ public void run() {
       final FileCoinArtifactSignerProvider fcArtifactSignerProvider =
           new FileCoinArtifactSignerProvider(fcBlsSignerProvider, fcSecpSignerProvider);
 
+      // create router from OpenApi spec - web3signer.yaml
```

Comment:
not sure if this is for now - but I'd love to see us split the configuration/buildup into Eth1, Eth2 and Filecoin "partitions" - thus the secp and the bls identifiers would never need to be shipped to a function together - they are fundamentally different, individual entities.